util/timer.h

```diff
@@ -42,23 +42,30 @@ class Timer {
         running_(false),
         executing_task_(false) {}
 
-  ~Timer() {}
```

Comment:
Curious to know the reason for the removal of destructor?

Reply:
It's an empty non-virtual destructor, I think it would be better to be removed. Or should we?